packages/lesswrong/lib/collections/posts/schema.js

```diff
@@ -386,8 +386,8 @@ const schema = {
   hiddenRelatedQuestion: {
     type: Boolean,
     viewableBy: ['guests'],
-    insertableBy: ['members'],
-    editableBy: [Users.owns, 'admins', 'sunshineRegiment'],
+    insertableBy: ['admins', 'sunshineRegiment'],
```

Comment:
Alas, this will break everything. Instead you want to set the `hidden` property to `true`. It's important that users can actually change this field on the backend.

Reply:
Out of curiosity, what's the mechanism of action here?

Reply:
The permission system. From the API perspective, it's the client that determines whether a question is a related question or not, via this field, so it's important that the client can send a graphQL request that actually inserts this field.